docker-compose.yml

```diff
@@ -1,7 +1,7 @@
 version: '3.7'
 services:
   zookeeper-1:
-    image: 'confluentinc/cp-zookeeper:${CONFLUENT_PLATFORM_VERSION:-7.0.1}'
+    image: 'confluentinc/cp-zookeeper:7.0.1'
```

Comment:
I don't expect Zookeeper version requirement to change as often as Kafka versions chan ge, so oped to hardcode Zookeeper version instead of adding an extra configuration parameter.